carbonserver/carbonserver/api/routers/authenticate.py

```diff
@@ -25,5 +25,6 @@ def auth_user(
         return Token(access_token="a", token_type="access")
     else:
         raise HTTPException(
-            status_code=status.HTTP_401_UNAUTHORIZED, detail="Incorrect password!"
+            status_code=status.HTTP_401_UNAUTHORIZED,
+            detail="Incorrect password or email!",
```

Comment:
In the following PR about errors management, should this be extracted in 2 different messages ?

Reply:
No, it is a security breach to tell if it is the password or the email that is wrong.
And it could be use to know if a person/email use the service.